app/models/users_role.rb

```diff
@@ -24,4 +30,14 @@ def self.current_role_for(user)
 
     nil
   end
+
+  # @param user [User]
+  # @param role [Role]
+  # @return [Boolean]
```

Comment:
Why do we need to return a boolean from this? Seems like it's only being used in specs, not in any application code.